Lib/ctypes/test/test_sizes.py

```diff
@@ -28,6 +28,9 @@ def test_size_t(self):
     def test_ssize_t(self):
         self.assertEqual(sizeof(c_void_p), sizeof(c_ssize_t))
 
+    def test_time_t(self):
+        self.assertIn(sizeof(c_time_t), (4, 8))
```

Comment:
Simple test case added.

As the reason for `c_time_t` is mostly to get the size of the object out of it, `test_sizes.py` is probably the best place to test for this. This test is mostly about checking for the existence of the field, and that it's 4 or 8. If/when platforms are found that have a differently-sized `c_time_t`, this test can be updated.

In the future, if we find a platform where `c_time_t` is of a different type (different size or even non-integer type), some other tests could be added.

Reply:
Why don't you compare `sizeof(c_time_t)` with `SIZEOF_TIME_T`?

Reply:
It seems like you should be able to use an independent version from `_testcapi`.

A